src/workerd/util/mimetype.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include <workerd/jsg/memory.h>
+#include <workerd/util/strings.h>
```

Comment:
Can this be moved to C++ so we can add it to implementation_deps?

Reply:
moved to c++?

Reply:
given that the file is used by multiple other locations that I'd rather not touch or impact, I'll hold off on this.